deployment/docker-compose/README.md

```diff
@@ -0,0 +1,463 @@
+
+    - Create the secret.  Only create one of the following secrets.
+        - jssecacerts - The java TrustStore file with any custom certificates imported.
+
+            ```cp <PATH_TO_TRUST_STORE_FILE> <PATH_TO_SECRETS>/jssecacerts```
+
+            or
+
+        - cacerts - The java TrustStore file with any custom certificates imported.
+
+            ```cp <PATH_TO_TRUST_STORE_FILE> <PATH_TO_SECRETS>/cacerts```
+
+### Insecure Trust of All Certificates
+WARNING: This is not a recommended option. Using this option makes your deployment less secure. Use at your own risk.
+Certificates SHOULD be correctly generated for the Alert server and a valid TrustStore SHOULD be provided to trust third party systems.
+
+This option allows the bypass of all certificate verification in the event that external certificates can not be imported into the Alert TrustStore, or certificate errors continue after importing the external certificates into the Alert TrustStore.
+
+To allow Alert to trust all certificates add the environment variable:
+```
+ALERT_TRUST_CERT=true
+```
+Please see the section [Environment Variables](#environment-variables) to learn how to set the environment variables.
+
+
+## Environment Variables
+Alert supports configuration of the application's components via environment variables.  There are two ways to configure the environment variables.
+1. Edit the blackduck-alert.env file.
+2. Edit the docker-compose.local-overrides.yml file to include the environment variables.
+
+Note: The docker-compose.local-overrides.yml file will need to be edited for other settings.  
+When installing edit either:
+
+```docker-compose.local-overrides.yml```
+
+or
+
+```blackduck-alert.env```
+
+### Edit Environment File
+Environment variables for Alert have already been created in this file but they are commented out.  
+Uncomment the variables to be set by deleting the '#' character at the beginning of each line and set its value.
+
+Example:
+```
+ALERT_HOSTNAME=localhost
+ALERT_LOGGING_LEVEL=INFO
+```
+
+### Edit the Overrides File
+Uncomment environment from the alert service section of docker-compose.local-overrides.yml.  
+Add environment variables as ```- <VARIABLE_NAME>=<VARIABLE_VALUE>``` into the ```environment: ``` section of the alert service.
+
+Example:
+```
+    environment:
+        - ALERT_HOSTNAME=localhost
+        - ALERT_LOGGING_LEVEL=INFO
+```
+
+### Environment Variable Overrides
+The environment variables will always take precedence and overwrite the values stored in the database if the following variable value is set to 'true'.
+
+```ALERT_COMPONENT_SETTINGS_SETTINGS_STARTUP_ENVIRONMENT_VARIABLE_OVERRIDE=true```
+
+### Alert Hostname Variable
+The ALERT_HOSTNAME environment variable must be specified in order for Alert to generate and use certificates correctly.
+- Add the ALERT_HOSTNAME environment variable the value must be the hostname only
```

Comment:
Need to add a comma between variable and the
```Add the ALERT_HOSTNAME environment variable, the value must be the hostname only```

Reply:
Or make them 2 different sentences

Reply:
I would actually put "the value must be the hostname only" in parentheses.

Reply:
`Add the ALERT_HOSTNAME environment variable. (The value must be the hostname only).`